src/components/partner/partner.service.ts

```diff
@@ -92,35 +89,37 @@ export class PartnerService {
     });
 
     const result = await this.repo.readOne(id, session);
+    return await this.secure(result, session);
+  }
 
-    const props = parsePropList(result.propList);
-    const secured = await this.authorizationService.secureProperties(
+  private async secure(dto: UnsecuredDto<Partner>, session: Session) {
+    const securedProps = await this.authorizationService.secureProperties(
       Partner,
-      props,
+      dto,
       session
     );
 
     return {
-      ...parseBaseNodeProperties(result.node),
-      ...secured,
-      modifiedAt: props.modifiedAt,
+      ...dto,
+      ...securedProps,
+      modifiedAt: dto.modifiedAt,
       organization: {
-        ...secured.organization,
-        value: result.organizationId,
+        ...securedProps.organization,
+        value: dto.organization,
       },
       pointOfContact: {
-        ...secured.pointOfContact,
-        value: result.pointOfContactId,
+        ...securedProps.pointOfContact,
+        value: dto.pointOfContact,
       },
       types: {
-        ...secured.types,
-        value: secured.types.value || [],
+        ...securedProps.types,
+        value: securedProps.types.value || [],
```

Comment:
These types, I think, are still required since they default the value to an empty array. This is a rough edge from our security layer that has yet to be handled.

So they need to be left as is.